content/en/docs/tasks/configure-pod-container/assign-resources/allocate-devices-dra.md

```diff
@@ -0,0 +1,310 @@
+
+   * `resource-driver.example.com/color` attribute with a value of `blue`
+   * `resource-driver.example.com/size` attribute with a value of `big`
+
+   The `constraints` field indicates that only devices that have an attribute
+   named `resource-driver.example.com/color` can be allocated to this
+   ResourceClaim.
+
+1. Create the ResourceClaim: 
+
+   ```shell
+   kubectl apply -f basic-dra-resourceclaim.yaml
+   ```
+
+For every available field, see the
+{{< api-reference page="workload-resources/resource-claim-v1beta2" text="ResourceClaim reference" >}}.
+
+### Create a ResourceClaimTemplate {#create-resourceclaimtemplate}
+
+Create a ResourceClaimTemplate if you want every Pod to have access to separate
+devices that have similar configurations. For example, you might want simultaneous
+access to devices for Pods in a Job that uses
+[parallel execution](/docs/concepts/workloads/controllers/job/#parallel-jobs).
+
+1. Save the following ResourceClaimTemplate as
+   `basic-dra-resourceclaimtemplate.yaml`:
+
+   ```yaml
+   apiVersion: resource.k8s.io/v1beta2
+   kind: ResourceClaimTemplate
+   metadata:
+     name: large-black-cat-claim-template
+   spec:
+     spec:
+       devices:
+         constraints:
+         - matchAttribute: "resource-driver.example.com/cat"
+         requests:
+         - name: large-black-cat-claim
+           deviceClassName: resource-driver.example.com
+           selectors:
+           - cel:
+              expression: |-
+                 device.attributes["resource-driver.example.com"].color == "black" &&
+                 device.attributes["resource-driver.example.com"].size == "large"
+   ``` 
+   This ResourceClaimTemplate defines a ResourceClaim that does the following
+   filtering:
+
+   * Selects devices that have the following attributes:
+       * `resource-driver.example.com/color` attribute with a value of `black`
+       * `resource-driver.example.com/size` attribute with a value of `large`
+   * Limits the devices that can be allocated to only devices that have the
+     `resource-driver.example.com/cat` attribute.
+
+1. Create the ResourceClaimTemplate: 
+
+   ```shell
+   kubectl apply -f basic-dra-resourceclaimtemplate.yaml
+   ```
+
+Kubernetes only generates ResourceClaims when you request this
+ResourceClaimTemplate in a workload.
+
+## Request devices in workloads using DRA {#request-devices-workloads}
+
+To request device allocation, specify a ResourceClaim or a ResourceClaimTemplate
+in the `resourceClaims` field of the Pod specification. Then, request a specific
+claim by name in the `resources.claims` field of a container in that Pod.
+You can specify multiple entries in the `resourceClaims` field and use specific
+claims in different containers.
+
+1. Save the following Job as `basic-dra-job.yaml`:
+
+   ```yaml
+   apiVersion: batch/v1
+   kind: Job
+   metadata:
+     name: job-with-cats
+   spec:
+     completions: 10
+     parallelism: 2
+     template:
+       spec:
```

Comment:
> The Job "job-with-cats" is invalid: spec.template.spec.restartPolicy: Required value: valid values: "OnFailure", "Never"

I don't think the particular value matters here, but it looks like one of them has to be defined here:

```suggestion
       spec:
         restartPolicy: Never
```